modules/specialized-guides/pages/large-deployments/hub-online-sync.adoc

```diff
@@ -0,0 +1,246 @@
+.. Field [literal]``Registration date`` shows the time when the peripheral server was registered to the hub server. 
+.. Field [literal]``Last modified`` shows the time of the last saved configuration change. 
+.. Field [literal]``Root Certificate Authority`` shows certificate details.
+   To download, edit or delete the root certificate, clicking btn:[Download], btn:[Edit] or btn:[Delete] respectively.
+   Deleting the certificate will break the connection between servers.
+.. Field [literal]``GPG Public Key`` shows whether the GPG key has been configured for the hub server.
+   For more information about GPG keys between hub and peripheral servers, see <<gpg-for-hub-online-sync>>.
+.. Field [literal]``Mirror credentials`` is the username the peripheral server uses when connecting to the hub server to synchronize vendor channels.
+   This username is generated automatically on the hub server, and then transmitted to the peripheral server during the registration phase.
+
+
+[[gpg-for-hub-online-sync]]
+==== GPG key usage with hub online synhronization
+
+When the metadata on the hub server are signed with a GPG key, the public key is automatically transmitted from hub to peripheral server.
+
+By default, {productname} is not signing metadata.
+Therefore, when the peripheral server is downloading data from the hub server there is no way of checking if the downloaded metadata have a valid signature, unless the customer has created their own GPG key.
+
+To enable checking of the data integrity, the GPG key needs to be created on the hub.
+When the peripheral server is configured to communicate with the hub, the public GPG key will then automatically be transferred to it.
+
+When the GPG key is created on the hub, field [literal]``GPG Public Key`` will be set to show that this server is using the GPG key.
+For more information about setting up own GPG key, see xref:administration:repo-metadata.adoc[].
+
+
+=== Deregister peripheral server
+
+Deregistration can happen from both sides, from the hub or from the peripheral server.
+
+.Procedure: Deregistering from the peripheral server
+. Go to menu:Hub Configuration[Hub Details]. 
+. Click btn:[Deregister].
+. Confirm the operation by clicking btn:[Deregister] on the pop-up window.
+. Page menu:Hub Configuration[Hub Details] is now empty.
+
+.Procedure: Deregistering from the hub server
+. Go to menu:Hub Configuration[Peripheral Configuration].
+. Find the perpheral server on the list.
+. Click btn:[Deregister] next to the peripheral server's name.
+. The peripheral server is no longer shown on the list.
+
+
+== Synchronize channels from hub to peripheral server
+
+Synchronizing vendor channels for the configured hub and server is done via dedicated user interface.
+
+.Procedure: Synchronizing channels from hub to peripheral server
+. Go to menu:Admin[Hub Configuration > Peripherals Configuration].
+. In the field [literal]``Synchronized channels`` click on btn:[Edit channels].
```

Comment:
I do not know where this button actually is. `field` sounds strange.  Maybe, "pane", "dialog", or "column"?  Or just:
```suggestion
. In [literal]``Synchronized channels`` click btn:[Edit channels].
```